src/components/TodoContext.tsx

```diff
@@ -0,0 +1,23 @@
+import { createContext } from 'react';
+
+type Props = {
+  isClearAllCompleted: boolean
+  isToggleAllCompleted: boolean
+  isToggleAllActive: boolean
+  inputValue: string,
+  todosGetter: () => void
```

Comment:
'todosGetter' seems to be a function to get todos. Consider renaming it to 'getTodos' to be more descriptive.

Reply:
Fixed